src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const isSelectedTab = tab.id === activeTab.id;
+
+            return (
+              <li
+                key={tab.id}
+                className={cn({ 'is-active': isSelectedTab })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => !isSelectedTab
+                    && onTabSelected(tab)}
```

Comment:
move to a handler function 
```
const onTabClick = (tab) => {
  if(tab.id === selectedTabId) {
    return
  }
  
  onTabSelect(tab)
}
```

```suggestion
                  onClick={() => onTabClick(tab)}
```